lib/settings/cubit/settings_state.dart

```diff
@@ -0,0 +1,9 @@
+class SettingsState {
```

Comment:
Should override equal and hash code, otherwise there could be issues.

Reply:
Added equitable